CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## 2.0.0-alpha.2
+- feat: Adds `upgrade` command to Serverpod CLI.
+- feat: Introduces `CacheMissHandler` to improve cache API.
+- fix: Maps Dart `int` to `bigint` in database.
+- fix: Generates thumbnails in isolates fo auth and chat module.
```

Comment:
Suggestion spelling mistake:
fix: Generates thumbnails in isolates **for** auth and chat module.